src/3d/qgspointcloudlayerchunkloader_p.cpp

```diff
@@ -47,10 +47,11 @@
 ///////////////
 
 QgsPointCloudLayerChunkLoader::QgsPointCloudLayerChunkLoader( const QgsPointCloudLayerChunkLoaderFactory *factory, QgsChunkNode *node, std::unique_ptr< QgsPointCloud3DSymbol > symbol,
-    double zValueScale, double zValueOffset )
+    double zValueScale, double zValueOffset, QgsCoordinateTransform coordTrans )
```

Comment:
can we make passing of the coordinate transform object more consistent? In some places it is passed by value, in others by non-const reference... best to pass it by const reference.